src/components/vuestic-components/vuestic-tabs/VuesticTabs.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="vuestic-tabs">
-    <div class="hidden-sm-down">
+    <div class="d-none d-sm-none d-md-flex">
```

Comment:
Hm, tabs are broken with this change..
@atalargo, could you please doublecheck?